content/movie-review-pt2.md

```diff
@@ -133,9 +98,9 @@ let rent_lamports = rent.minimum_balance(account_len);
 
 Before creating an account, we also need to have an address to assign the account. For program owned accounts, this will be a program derived address (PDA) found using the `find_program_address` function. As the name implies, PDAs are derived using the program Id (address of the program creating the account) and an optional list of “seeds”. Optional seeds are additional inputs used in the `find_program_address` function to derive the PDA. Providing additional seeds allows us to create an arbitrary number of PDA accounts and a deterministic way to find each account.
 
-There is one more seed that is needed to derive a PDA. This seed is referred to as the “bump seed” and is a number between 255-0. It is a requirement that PDAs do not have a corresponding private key. This is done by deriving the PDA using a new “bump seed” until a valid PDA is found. The `find_program_addres` function tries to find a PDA using the optional seeds provided, the program ID, and the “bump seed” starting from 255. If the output is not a valid PDA, then the function decreases the bump by 1 and tries again (255, 254, 253, etc). Once a valid PDA is found, the function returns both the PDA and the bump that was used to derive the PDA.
+In addition to the seeds you provide for deriving a PDA, the `find_program_adress` function will provide one additional "bump seed." What makes PDAs unique from other Solana account addresses is that they do not hae a corresponding secret key. This ensures that only the program that owns the address can sign on behalf of the PDA. When the `find_program_addres` function attempts to derive a PDA using the provided seeds, it passes in the number 255 as the "bump seed." If the resulting address is invalid (i.e. has a corresponding secret key), then the function decreases the bump seed by 1 and derives a new PDA with that bump seed. Once a valid PDA is found, the function returns both the PDA and the bump that was used to derive the PDA.
 
-For our note-taking program, we will use the note creator's publickey and the id as the optional seeds to derive the PDA. Deriving the PDA this way allows us to deterministicly find the account for each note.
+For our note-taking program, we will use the note creator's public key and the ID as the optional seeds to derive the PDA. Deriving the PDA this way allows us to deterministicly find the account for each note.
```

Comment:
typo "deterministicly" --> deterministically

Reply:
Wow haha that's a bad one